packages/design-system/src/types/Table/TableCaption.d.ts

```diff
@@ -23,6 +23,6 @@ export interface TableCaptionProps {
   _scrollableNotice?: React.ReactNode;
 }
 
-declare const TableCaption: React.FC<TableCaptionProps>;
+declare const TableCaption: React.FC<React.HTMLProps<TableCaptionProps>>;
```

Comment:
This doesnt look right, you are passing in our own defined props into `React.HTMLProps`

Reply:
Thanks for picking that up. Fixed it